src/utils_request.js

```diff
@@ -27,7 +27,8 @@ export const REQUEST_AS_BROWSER_DEFAULT_OPTIONS = {
  * Currently, the function sends requests the same way as Firefox web browser does.
  * In the future, it might add support for other browsers too.
  *
- * Internally, the function uses `@apify/httpRequest` function to perform the request.
+ * Internally, the function uses `httpRequest` function from the
+ * [`@apify/httpRequest`](https://github.com/apifytech/http-request) to perform the request.
```

Comment:
The docs need to be precise if people are to take the library seriously:

Internally, the function uses `httpRequest` function from the [`@apify/http-request`](https://www.npmjs.com/package/@apify/http-request) NPM package to perform the request.